tests/models/test_teacher_forcing_logits.py

```diff
@@ -6,6 +6,39 @@
 import shap
 
 
+def test_falcon():
+    transformers = pytest.importorskip("transformers")
+    requests = pytest.importorskip("requests")
+    name = "fxmarty/really-tiny-falcon-testing"
+    try:
+        tokenizer = transformers.AutoTokenizer.from_pretrained(name)
+        model = transformers.AutoModelForCausalLM.from_pretrained(
+            name, trust_remote_code=True, load_in_8bit=True, device_map="auto"
+        )
+    except requests.exceptions.RequestException:
+        pytest.xfail(reason="Connection error to transformers model")
```

Comment:
Great idea

Reply:
just stole this from the other test in the file ;) But thanks I'll take the compliment